nile-backend/src/main/java/com/tp/Nile/config/SecurityConfig.java

```diff
@@ -97,13 +97,20 @@ protected void configure(HttpSecurity http) throws Exception {
                         "/api/specs/**",
                         "/api/types/**",
                         "/api/vendors/**",
+
+
+                        "/api/productPhotos/**",
```

Comment:
Duplicate path here and line 108 it looks like

Reply:
good catch